### ISSUE TITLE
while using HighChart version 11.2 I Facing an issue on tooltip that displays ?

### ISSUE THREAD

**Issue by vanshdeepsinha** [highcharts/highcharts#20438]
while using highchart version 11.2 I am facing issue of ? on the tooltip during loading bar graph, please find the following code below

Highcharts.chart(chartId, {
		chart: chartattr,
		colors: graphColor,
		plotOptions: varPlotOptions,
		title: {
			text: " "
		},
		xAxis: {
			title: {
				style: {
					color: graphFontcolor,
					fontSize: XAxisFontSize
				},
				text: json.xAxisLabel,

			},
			max: maxValueOfAxis,
			min: minValueOfAxis,
			categories: ticks,
			labels: {
				style: {
					color: graphFontcolor,
					fontSize: XAxisFontSize,
					textShadow: 'none'
				}
			}
		},

		yAxis: {
			title: {
				style: {
					color: graphFontcolor,
					fontSize: YAxisFontSize
				},
				text: json.yAxisLabel
			},

			labels: {
				style: {
					color: graphFontcolor,
					fontSize: YAxisFontSize,
					textShadow: 'none'
				}
			},
			min: 0

		},
		legend: legendConfiguration,
		
		tooltip: {
			pointFormat: pointFormat,
			shared: true
		},
		series: eval('(' + arrseries + ')'),
		credits: false,
		exporting: {

			chartOptions: { // specific options for the exported image
				title: {
					text: rptName
				},
				subtitle: {
					text: currdt,
					align: "right"
				},
				credits: {
					position: {
						align: 'left',
						y: -20,
						x: 10
					},
					text: text,
					href: null,
					style: {
						color: "#000000",
						fontSize: "9px"
					}

				},

				scrollbar: { enabled: false },
				legend: {
					
					y: -40,
					floating: false,
					borderWidth: 1,
					
					shadow: true,
					itemStyle: {
						fontSize: 10
					}

				},//added
				plotOptions: {
					series: {
						dataLabels: {
							enabled: true,
							style: {
								textShadow: 'none'
							}
						}
					}
				}
			},
			scale: 1,
			fallbackToExportServer: false
		},
		scrollbar: scrollbar

	});
}

![graphError](https://github.com/highcharts/highcharts/assets/48940719/1350acac-67a4-45ad-866c-a71090902ed5)

**Comment by vanshdeepsinha** [comment]
yes..!! I did change pointFormat to Formatter : pointFormat as per the guidelines but nothing changed/happened still the same issue

**Comment by karolkolodziej** [comment]
Could you please then recreate the issue in some online code editor so I can work on it?

**Comment by vanshdeepsinha** [comment]
Hi

I have resolved the issue myself by updating tooltip option like 
tooltip: {
        formatter: function () {
            return this.x +'</b>:<b>' + this.y + '</b>';
  } } , ....
  
I tested this code with LineGraph and then updated the same in all the other graphs like BarGraph and PieChart as per requirement to display the data

Thanks a Lot
